samples/sample-kotlin-android/build.gradle.kts

```diff
@@ -124,6 +124,8 @@ dependencies {
     implementation("androidx.compose.runtime:runtime-livedata:1.4.3")
     implementation("androidx.compose.ui:ui-tooling-preview:1.4.3")
     implementation("androidx.lifecycle:lifecycle-runtime-ktx:2.6.2")
+    // adding play services to generate advertising id
+    implementation("com.google.android.gms:play-services-ads:22.1.0")
```

Comment:
👍  Great addition to the sample application!! 

But why are we adding an old version of `com.google.android.gms:play-services-ads`?
The latest version is: 
`com.google.android.gms:play-services-ads:23.1.0`